flink-streaming-java/src/main/java/org/apache/flink/streaming/api/functions/source/ContinuousFileReaderOperator.java

```diff
@@ -292,6 +294,7 @@ public void run() {
 						// close and prepare for the next iteration
 						this.format.close();
 						this.currentSplit = null;
+						completedSplitsCounter.inc();
```

Comment:
Should we move the increment out of the `finally` clause to avoid counting a failed split as processed?